packages/generator-common/src/compiler.ts

```diff
@@ -50,8 +53,39 @@ export async function transpileDirectory(
     allFiles2.map(file => resolve(path, file)),
     compilerOptions
   );
-  const emitResult = await program.emit();
 
+  // The write file handler does not support async function hence the work around with the outer promise list.
+  const fileWriterPromises: Promise<void>[] = [];
+  const prettierWriter: WriteFileCallback = (
+    fileName,
+    text,
+    writeByteOrderMark,
+    onError?,
+    sourceFiles?,
+    data?
+  ) => {
+    const parsed = parse(fileName);
+    const promise = mkdir(parsed.dir, { recursive: true }).then(() => {
+      // All not emitted files like .md or .json should be already prettified.
+      // Formatting emitted .js file could break source map -> skip these.
+      // Formatting the .map files also is not necessary increases only file size - only .d.ts files are nicer if linted
+      const usePrettier = getFileExtension(fileName) === 'd.ts' ? true : false;
+
+      // Prettier config should be cached from previous steps already.
+      // Emit also overwrites and copyright is either already there.
```

Comment:
[req] I do not understand this sentence ;) Can you rephrase it?

Reply:
removes because nonsense.